lib/normalize.js

```diff
@@ -34,6 +34,8 @@ module.exports = function normalize(nemo, _locator) {
   var normalizedLocator;
   if (_locator.constructor === String) {
     locator = _splitLocator(nemo, _locator);
+  } else if (locator instanceof nemo.wd.By.constructor) {
+      return locator; //no need to normalize, WebDriver understands this format.
   }
```

Comment:
Is there another use case? because the above would better be written as:
`nemo.view.simple.waitButton()`